app/helpers/orders_helper.rb

```diff
@@ -54,4 +54,35 @@ def expired_class(order)
       ''
     end
   end
+
+  def prepare_order_list(orders)
```

Comment:
Just moved these methods to OrdersHelper because ApplicatioHelper is too long now and these methods are about orders list actually.